packages/react-server/README.md

```diff
@@ -5,3 +5,216 @@ This is an experimental package for creating custom React streaming server rende
 **Its API is not as stable as that of React, React Native, or React DOM, and does not follow the common versioning scheme.**
 
 **Use it at your own risk.**
+
+## Usage
```

Comment:
It would be cool to include a `Reference` section with the exports for flight and fizz (even with just the types).

Reply:
added a partial reference.